src/main/java/com/ldtteam/structurize/util/BlockUtils.java

```diff
@@ -264,6 +265,7 @@ public static boolean isWater(final BlockState iBlockState)
         return iBlockState.getBlock() == Blocks.WATER;
     }
 
+    @Deprecated(forRemoval = true, since = "1.21")
     private static Item getItem(final BlockState blockState)
```

Comment:
we will still need solutions for this kind of things, don't just deprecate it =D

Reply:
It's in blockui for two version already